.github/ISSUE_TEMPLATE/package_improvement.yml

```diff
@@ -1,4 +1,4 @@
-name: Package improvements 🛠
+name: Package improvement 🛠
```

Comment:
A quick follow-up to #2200 & #2360 that doesn't need its own pull request.

Reply:
👍